packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.base.tsx

```diff
@@ -457,7 +458,7 @@ export class DetailsListBase extends BaseComponent<IDetailsListProps, IDetailsLi
                   onItemInvoked={onItemInvoked}
                   onItemContextMenu={onItemContextMenu}
                   enterModalOnTouch={this.props.enterModalSelectionOnTouch}
-                  {...selectionZoneProps || {}}
+                  {...(selectionZoneProps || {})}
```

Comment:
Good catch 👍

Reply:
That change was made automatically by prettier!